### ISSUE TITLE
Update Docs For Computational Performance

### ISSUE THREAD

**Issue by AlexSchuy** [KevinMusgrave/pytorch-metric-learning#194]
See #192. For other people who encounter performance problems, perhaps a 'performance optimization' section can be added to the docs that describes first using a miner of [type 2](https://github.com/KevinMusgrave/pytorch-metric-learning/issues/192#issuecomment-689814355) and either chaining the results into another miner or into a loss function.
